recipes/twitch-native-ipc/all/CMakeLists.txt

```
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.1)
+project(cmake_wrapper)
+include(conanbuildinfo.cmake)
+
+list(APPEND CONAN_BASIC_SETUP_ARGS TARGETS NO_OUTPUT_DIRS)
+
+if(APPLE)
+  list(APPEND CONAN_BASIC_SETUP_ARGS KEEP_RPATHS)
+endif()
+
+conan_basic_setup(${CONAN_BASIC_SETUP_ARGS})
```

- `KEEP_RPATHS` is unwanted for conan recipes, because the rpath on c3i is different than the consumers.
- `NO_OUTPUT_DIRS` is only needed when some `add_custom_command/add_custom_target/add_test` uses relative paths instead of targets.
- `TARGETS` is nice to have, but not absolutely required.

```suggestion
conan_basic_setup()
```